src/app/meshes/views/MeshConfigView.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <RouteView
+    v-slot="{ route }"
+    name="mesh-config-view"
+    data-testid="mesh-config-view"
+  >
+    <AppView>
+      <template #title>
+        <h2>
+          <RouteTitle
+            :title="t('meshes.routes.item.navigation.mesh-config-view')"
+            :render="true"
+          />
+        </h2>
+      </template>
+
+      <KCard class="mt-4">
+        <template #body>
+          <DataSource
+            v-slot="{ data, error }: MeshSource"
+            :src="`/meshes/${route.params.mesh}`"
```

Comment:
Is this another place where we are loading the same data everytime we click a tab?

Anywhere we use the same data across different tabs, the DataSource should be added in the topmost View (where the tabs are) and then passed down using `RouterView`.

This means we make far fewer HTTP requests and only refresh the data when you first land on the 'detail/tabbed' page.

Are there other places where we are loading the same data over and over everytime you click a tab?

Reply:
It’s not easy to actually have such a clear-cut rule because it’s not always the case that all tabs in such a view need access to the same data. In this case, the Mesh resource is only needed when accessing the Overview or YAML tabs. Moving this data source into the tabs view component would cause it to be loaded if you, for example, access services directly without even going to the Mesh overview.

Reply:
Oh gotcha, interesting one, I guess we could only add the datasource to the page if certain tabs are active?

Reply:
Yeah, we could build something like that, but it’s kind of strange to transfer the knowledge of which "deep" view needs which data into a parent view (i.e. the tabs view). Not really sure how to handle this better.

Reply:
Not sure how thats strange, we quite often pass data down from parent components into deeper components

Reply:
Is this something you are looking at, or do we want to do that in a separate PR?

Reply:
Not looking at this, no. I don’t think in this case, moving the data source into the tabs is appropriate so I also don’t plan on following up on this immediately. This is fine.

Reply:
Ok I think we are good with this for the minute then, one sec